### ISSUE TITLE
SA1019: httputil.ClientConn has been deprecated since Go 1.0: Use Client or Transport in package net/http instead.

### ISSUE THREAD

**Issue by EtienneM** [Scalingo/cli#919]
```
apps/run.go:350:12: SA1019: httputil.ClientConn has been deprecated since Go 1.0: Use Client or Transport in package net/http instead. (staticcheck)                                          
        var conn *httputil.ClientConn                                                                                                                                                         
                  ^                                                                            
apps/run.go:356:10: SA1019: httputil.NewClientConn has been deprecated since Go 1.0: Use the Client or Transport in package net/http instead. (staticcheck)                                   
                conn = httputil.NewClientConn(tlsConn, nil)                                                                                                                                   
                       ^                                                                                                                                                                      
apps/run.go:358:10: SA1019: httputil.NewClientConn has been deprecated since Go 1.0: Use the Client or Transport in package net/http instead. (staticcheck)                                   
                conn = httputil.NewClientConn(dial, nil)                                                                                                                                      
```

**Comment by EtienneM** [comment]
I tried to find a solution but didn't succeed. There is no more HTTP client hijack in Go (https://github.com/golang/go/issues/28030). Hence we need to figure out another way to make the underlying TCP connection bidirectional.

**Comment by briceamen** [comment]
Hi @EtienneM,

I've created a [draft PR](https://github.com/Scalingo/cli/pull/1090) addressing the httputil.ClientConn deprecation.

I've tested this successfully locally and on staging through `run bash` is there any other one-off situation that I should test ? 

Moreover I wasn't able to add unit tests yet due to issues importing the run service mock. I'll need some help with the mocking setup to complete the test coverage.

Let me know what you think about this approach!
